addons/hr_holidays/models/hr_employee.py

```diff
@@ -158,6 +158,8 @@ def _compute_leave_manager(self):
             manager = employee.parent_id.user_id
             if manager and employee.leave_manager_id == previous_manager or not employee.leave_manager_id:
                 employee.leave_manager_id = manager
+            elif not employee.leave_manager_id:
+                employee.leave_manager_id = False
```

Comment:
I suppose that it's to set a value in all the cases, but it seems louche.
Why not:
``` python
else:
    employee.leave_manager_id = False
```